### PR TITLE
More TS 3.2 DOM updates

### PR DESCRIPTION
**Pull request by sandersn**
Unlike the other PR, these updates should not go in until 3.2 is out, since they rely on the following additions:

1. webrtc
2. web-animations
3. geometry-dom

I left these packages in to make it easier to upgrade and edited them where needed, but I removed the dependencies on them in other Definitely Typed packages. I also left them in because we don't have a good way to mark a *maximum* version of Typescript that a package applies to. The edits I made make sure these packages at least do not fail in 3.2 when you install them.

Please do NOT merge this PR until after Typescript 3.2 is released.

### CODE REVIEW

**Comment by sandersn** [comment]
Remaining CI failures are because the header parser doesn't recognise 3.2 yet.

File: types/peerjs/index.d.ts [hunk 1]
```diff
@@ -2,9 +2,7 @@
 // Project: http://peerjs.com/
 // Definitions by: Toshiya Nakakura <https://github.com/nakakura>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
-
-/// <reference types="webrtc" />
+// TypeScript Version: 3.2
```
Comment (Kovensky) [inline types/peerjs/index.d.ts]: I thought you had to use `typesVersion` when using any version `>= 3.1`

Comment (sandersn) [reply types/peerjs/index.d.ts]: No, that's only if you want to provide simultaneous 3.1+ and pre-3.1 support.

Comment (sandersn) [reply types/peerjs/index.d.ts]: Later: Well, looks like I'm wrong! The build fails when you require a version newer than 3.1 now. I guess I'll have to learn how to use typesVersions again.

**Pull request merged**